scenarios/classification/25_deployment_on_azure_apps_service_and_setting_CORS_policies.ipynb

```diff
@@ -0,0 +1,1561 @@
+{
```

Comment:
<p>Since we are assuming the 21_notebook was executed and are getting the Model 'im_classif_resnet18'. Should we also assume that script.py already exists and remove the description and code here? Would make the notebook smaller.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/533/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I added this as an optional step to make it a bit more user friendly and for the notebook to be a bit more self contained, but I think it's a fair point.</p><p> </p><p>Also, I noticed that it may make sense for us to review and clean up our notebooks as many reference each other and the resources created from them.</p><p> </p><p>I'm considering refactoring the creation of the common resources and components into a separate notebook that should be run before running the individual notebooks so that all the following notebooks can assume that the common resources (such as the scoring script) already exist and can point users to a single notebook for resource creation.</p><p> </p><p>This will also make it easier for us to manage the creation of the common resources as we will have a single reference notebook to add steps to create new resources required for notebooks.</p>